src/api/v1/notifications/handlers/handle-xmtp-notification.ts

```
@@ -0,0 +1,317 @@
+        where: {
+          pushToken: notification.installation.delivery_mechanism.token,
+        },
+        include: {
+          identities: {
+            include: {
+              identity: true,
+            },
+          },
+        },
+      });
+      if (device && device.expoToken) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)